docs/modules/migrate/pages/rolling-restart.adoc

```diff
@@ -0,0 +1,45 @@
+= Upgrade Cluster to {enterprise-product-name}
+:description: You can use an in-place rolling restart to move each member on your Open Source cluster to an Enterprise cluster.
+
+{description}
+
+This approach ensures that you can move your cluster without interruption to the service. 
+Similar to the rolling upgrade process, a rolling restart allows you to move one member at a time.
+
+The codebase of your {open-source-product-name} cluster must match the target {enterprise-product-name} release. 
+Currently, rolling restart can only be used to update your 5.5 {open-source-product-name} cluster members.
+
+== Best Practice
+
+Before starting a rolling restart, consider the following best practices:
+
+* Backup your IMap and ReplicatedMap data structures. This is not essential, but it is recommended practice
+* If the {open-source-product-name} member is a different version to the target {enterprise-product-name} member, update the codebase to match the target {enterprise-product-name} release
++
+**[RJL: Do we need to include any additional information to cover moving from - for example - 5.3 with CP Subsystem to Enterprise 5.5?]**
+
+* Ensure that the version of JDK meets the minimum required for the target {enterprise-product-name} release
+* Plan sufficient time for the rolling restart to complete, and remember that the member cannot change during the move
+
+== Perform a Rolling Restart
+
+To move your {open-source-product-name} cluster to {enterprise-product-name}, complete the following steps:
+
+. Gracefully shut down an existing {open-source-product-name} member.
++
+See xref:maintain-cluster:shutdown.adoc#shutting-down-a-hazelcast-member[Shutting Down a Hazelcast Member].
+
+. Wait for all partitions to migrate to the rest of the cluster.
++
+During migrations, members cannot join or leave the cluster.
+
+. Upgrade the member's codebase to match the {enterprise-product-name} version.
+
+. Start the member and wait until it joins the {enterprise-product-name} cluster.
+
+. Repeat this process for each member in the {open-source-product-name} cluster.
+
+. When all members on the {open-source-product-name} cluster have been moved to the {enterprise-product-name} cluster, enable and configure the 
```

Comment:
I would link dynamic configuration here. It's the only well supported way to enable EE features on a running cluster.